content/00.front-matter.md

```diff
@@ -21,9 +21,12 @@ on {{date}}.
 {% if author.orcid is defined -%}
 [![ORCID icon](images/orcid.svg){height="11px" width="11px"}](https://orcid.org/{{author.orcid}})
 {%- endif %}
-{{author.name}}<sup>{{author.affiliation_numbers | join(',')}}</sup>
+{{author.name}}<sup>{{author.affiliation_numbers | join(',')}}{%- if author.symbol_str is defined -%},{{author.symbols | join(',')}}{%- endif -%}</sup>
 {%- if not loop.last -%}, {%- endif -%}
 {% endfor %}
+
+<sup>☯</sup> --- Author order was determined with a randomized algorithm<br>
+<sup>†</sup> --- To whom correspondence should be addressed: `gitter` at `biostat.wisc.edu` (AG) and `csgreene` at `upenn.edu` (CSG)
```

Comment:
This workaround looks good to me.  We could use `gitter at biostat.wisc.edu` instead of `gitter` at `biostat.wisc.edu`, but I don't have a strong preference.